.travis.yml

```diff
@@ -62,14 +68,23 @@ script:
   - curl https://raw.githubusercontent.com/jhipster/generator-jhipster/master/test-integration/samples/.jhipster/Label.json -o .jhipster/Label.json
   - curl https://raw.githubusercontent.com/jhipster/generator-jhipster/master/test-integration/samples/.jhipster/Operation.json -o .jhipster/Operation.json
 
-  - npm link generator-jhipster
   - npm link generator-jhipster-helloworld
   - jhipster --blueprint helloworld --force --no-insight
 
+  # link again generator-jhipster, because of npm install
```

Comment:
a bug I encounter each times there is an npm install, I need to do a npm link again